libevmasm/AssemblyItem.cpp

```diff
@@ -241,6 +243,9 @@ string AssemblyItem::toAssemblyText(Assembly const& _assembly) const
 	case UndefinedItem:
 		assertThrow(false, AssemblyException, "Invalid assembly item.");
 		break;
+	case VerbatimData:
+		text = string("verbatimdata_") + to_string(m_verbatimData->first) + "_" + util::toHex(m_verbatimData->second);
```

Comment:
Why do we need the stack diff (`->fist`) here? Also while technically legal, `_-1` looks unusal.

Reply:
Not sure we need it